tmp_src/sass/_footer.scss

```diff
@@ -44,7 +44,8 @@
   font-size: .875em;
 }
 
-.footer-licensing a {
+.footer-licensing a,
+.footer-licensing__a {
```

Comment:
I'm curious, sorry if that’s a stupid question I miss the context entirely, but why do we do that?